<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTKPluginGeneratorMainExtension</class>
 <widget class="QWidget" name="QTKPluginGeneratorMainExtension">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="2">
    <widget class="QLineEdit" name="symbolicNameEdit"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="versionEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Symbolic name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Version</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Activation policy</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="activationPolicyCombo">
     <item>
      <property name="text">
       <string>Lazy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Eager</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QWidget" name="advancedButton" native="true">
     <property name="text" stdset="0">
      <string>Advanced</string>
     </property>
     <property name="collapsed" stdset="0">
      <bool>true</bool>
     </property>
     <property name="collapsedHeight" stdset="0">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Export directive</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="exportDirectiveEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Activator class name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="activatorClassEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Activator header file</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="activatorHeaderEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Activator source file</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="activatorSourceEdit"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
